storage/core/src/server/ServerStorage.ts

```diff
@@ -112,22 +112,34 @@ export abstract class ServerStorage
 
   public abstract getDownloadUrl(
     reference: ObjectReference,
-    expiresInSeconds?: number
+    options?: {
```

Comment:
I think `expiry.expiresInSeconds` is fine, and I would still prefer `expiry` because `options` to me is pretty vague and could be used to group not-that-related possibly optional parameters, here we explicitly want to get expiry information in one form or the other.